src/mcpdotnet/Protocol/Types/JsonSchema.cs

```diff
@@ -10,7 +10,7 @@ public class JsonSchema
     /// The type of the schema, should be "object".
     /// </summary>
     [System.Text.Json.Serialization.JsonPropertyName("type")]
-    public string Type { get; set; } = string.Empty;
+    public string Type { get; set; } = "object";
```

Comment:
Nit: consider making a pure DTO
```suggestion
    public required string Type { get; set; }
```

Reply:
I'm going to leave it without the required for now. I think all of these types need to be revisited.